orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -163,7 +158,7 @@ struct OrtTrainingApi {
   * throughout the training session.
   * Please note that this function does not set the initial learning rate that may be needed
   * by the predefined learning rate schedulers. To set the initial learning rate for learning
-  * rate schedulers, please look at the function `RegisterLRScheduler`.
+  * rate schedulers, please look at the function `RegisterLinearLRScheduler`.
```

Comment:
a suggestion: If we want to change this, maybe it's better we split them from this PR?

Reply:
It will be incomplete to skip the LR scheduler in the C# bindings, and might take more effort than necessary to separate the parts out.